osu.Game/Overlays/Options/SliderOption.cs

```diff
@@ -104,13 +105,14 @@ public OsuSliderBar()
                         AutoSizeAxes = Axes.None,
                         RelativeSizeAxes = Axes.None,
                         Masking = true,
-                        BorderColour = new Color4(255, 102, 170, 255),
+                        BorderColour = OsuColor.SliderbarNub,
                         BorderThickness = 3,
                         Children = new[]
                         {
                             new Box
                             {
-                                Colour = new Color4(255, 102, 170, 0),
+                                Colour = new Color4(OsuColor.SliderbarNub.R,

```

Comment:
can you add an explanation whenever you use "-1" in a sentence? it's pointless on its own.

also if it's called Transparent the default should be 1 which is "fully transparent"

if 0 is transparent then the function should be called Opacity